jams/eval.py

```diff
@@ -479,3 +480,50 @@ def pattern(ref, est, **kwargs):
     est_patterns = pattern_to_mireval(est)
 
     return mir_eval.pattern.evaluate(ref_patterns, est_patterns, **kwargs)
+
+
+def transcription(ref, est, **kwargs):
+    r'''Note transcription evaluation
+
+    Parameters
+    ----------
+    ref : jams.Annotation
+        Reference annotation object
+    est : jams.Annotation
+        Estimated annotation object
+    kwargs
+        Additional keyword arguments
+
+    Returns
+    -------
+    scores : dict
+        Dictionary of scores, where the key is the metric name (str) and
+        the value is the (float) score achieved.
+
+    See Also
+    --------
+    mir_eval.transcription.evaluate
+
+    Examples
+    --------
+    >>> # Load in the JAMS objects
+    >>> ref_jam = jams.load('reference.jams')
+    >>> est_jam = jams.load('estimated.jams')
+    >>> # Select the first relevant annotations. You can use any annotation
+    >>> # type that can be converted to pitch_hz (such as pitch_midi)
+    >>> ref_ann = ref_jam.search(namespace='pitch_hz')[0]
+    >>> est_ann = est_jam.search(namespace='pitch_midi')[0]
+    >>> scores = jams.eval.transcription(ref_ann, est_ann)
+    '''
+
+    namespace = 'pitch_hz'
+    ref = coerce_annotation(ref, namespace)
+    est = coerce_annotation(est, namespace)
+    ref_intervals, ref_pitches = ref.data.to_interval_values()
+    est_intervals, est_pitches = est.data.to_interval_values()
+
+    ref_pitches = np.array(ref_pitches)
```

Comment:
Why is it necessary to call `array` on this?  `to_interval_values()` already returns numpy arrays.